packages/desktop-electron/preload.ts

```diff
@@ -1,16 +1,27 @@
-const { ipcRenderer, contextBridge } = require('electron');
+import { ipcRenderer, contextBridge, IpcRenderer } from 'electron';
 
-const { version: VERSION, isDev: IS_DEV } =
+import {
+  GetBootstrapDataPayload,
+  OpenFileDialogPayload,
+  SaveFileDialogPayload,
+} from './index';
+
+const { version: VERSION, isDev: IS_DEV }: GetBootstrapDataPayload =
   ipcRenderer.sendSync('get-bootstrap-data');
 
 contextBridge.exposeInMainWorld('Actual', {
   IS_DEV,
   ACTUAL_VERSION: VERSION,
-  logToTerminal: (...args) => {
-    require('console').log(...args);
+  logToTerminal: (...args: unknown[]) => {
+    console.log(...args);
   },
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
  logToTerminal: console.log,
```